packages/firestore/test/unit/specs/spec_test_runner.ts

```diff
@@ -549,6 +549,16 @@ abstract class TestRunner {
   async run(step: SpecStep): Promise<void> {
     await this.doStep(step);
     await this.queue.drain();
+    // Wait for any additional operations that the initial operations
+    // scheduled. We skip this iterative drain for steps that involve changing
```

Comment:
If you can phrase it naturally, try to avoid "we" in comments about the system. Sometimes it can't be avoided--don't worry about it too much.

In this case using this form is placing unnecessary emphasis on us as the authors when the real actors here are the operations that might change the primary tab. Consider rephrasing to put more emphasis on that. Something like:

> However, when the current step changes the primary tab, the additional operations are expected by later steps, so avoid draining until empty in this case.

Reply:
thanks for the explanation -- that makes a lot of sense